src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -798,26 +755,27 @@ public void LoadGeneric6(InputType inputType, ReaderType readerType)
         [Theory]
         public void LoadGeneric7(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
         {
-            AppContext.SetSwitch("Switch.System.Xml.AllowDefaultResolver", true);
-
-            string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
+            using (new AllowDefaultResolverContext())
+            {
+                string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
 
-            FileStream s2;
+                FileStream s2;
 
-            // check immediately after load and after transform
-            if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
-            {
-                s2 = new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read);
-                s2.Dispose();
-                if (Transform("fruits.xml", transformType, docType) == 1)
+                // check immediately after load and after transform
+                if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
                 {
-                    VerifyResult(expected);
                     s2 = new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
what's the purpose of this? do we check if file is locked? also this could be `new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read).Dispose()` or `using (new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read));` we could avoid using the variable - also some comment would be helpeful